dev/raytracing/shaders/fragment.glsl

```diff
@@ -691,10 +702,12 @@ material_params(RayHit ray_hit)
         int index = 6 * ray_hit.tet_num + ray_hit.object_index;
         int color_index = edge_color_indices[index];
         
-        result.diffuse =
-            vec3(0.5, 0.5, 0.5)
-            + sin(color_index) * vec3( 0.3,  -0.3,   0.0)
-            + cos(color_index) * vec3(0.15,   0.15, -0.3);
+        // result.diffuse =
+        //     vec3(0.5, 0.5, 0.5)
+        //     + sin(color_index) * vec3( 0.3,  -0.3,   0.0)
+        //     + cos(color_index) * vec3(0.15,   0.15, -0.3);
+        result.diffuse = hsv2rgb(vec3(float(color_index)/float(num_tets), 1.0, 1.0));
+        // num_tets = num_edges
```

Comment:
Move up one line and write "using num_tets = num_edges" so that it is clear that this is not code that was commented.

Reply:
Done.